script.js

```diff
@@ -0,0 +1,98 @@
+let cardObjectsArray = [
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/pinkHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/pinkHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/wheelHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/wheelHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/yellowHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/yellowHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/coralHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/coralHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/greenHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/greenHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/tealHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/tealHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/handHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/handHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/smokingHamster.jpg'
+    },
+    {
+        backOfCard : 'images/cardBack.jpg',
+        frontOfCard : 'images/smokingHamster.jpg'
+    }
+]
+
+function shuffleCards(array) {
+    let startingIndex = array.length;
+    let temporaryIndex = 0;
+    let randomIndex = 0;
+
+    while (startingIndex !== 0) {
+        randomIndex = Math.floor(Math.random() * startingIndex);
+        startingIndex -= 1;
+        temporaryIndex = array[startingIndex];
+        array[startingIndex] = array[randomIndex];
+        array[randomIndex] = temporaryIndex;
+    }
+    return array;
+}
+
+let shuffledArrayOfIndexes = shuffleCards(cardObjectsArray)
+let cardDivArray = document.querySelectorAll('.card')
+i = 0;
+cardDivArray.forEach(card => {
+    card.innerHTML = '<img class="hamster" src="' + shuffledArrayOfIndexes[i].frontOfCard + '">'
+        + '<img class="sloth" src="' + shuffledArrayOfIndexes[i].backOfCard + '">';
+    console.log(i)
+    return i++;
```

Comment:
returning outside of a function?? just `i++` without `return` would achieve the effect you want

Reply:
instead of this way of doing it with `i` and `forEach`, use a `for` loop:
```js
for(i = 0; i < cardDivArray.length; i++) {
   // do the stuff here and use `i` as the index for both `cardDivArray` and `shuffledArrayOfIndexes`
}
```
also, remove `console.log(i)`